lib/showoff_utils.rb

```diff
@@ -68,7 +68,9 @@ def self.create(dirname,create_samples,dirs='one')
 
   def self.skeleton(config)
     if config
-      FileUtils.cp(config, '.')
+      unless File.expand_path(config) == File.expand_path(File.basename(config),'.')
```

Comment:
Would `unless File.exist?(File.join('.', File.basename(config))` read intent more clearly?

Reply:
@binford2k No, that does something different (and probably not desirable).  My intent was only to avoid attempting to `FileUtils.cp` a file to its own path (which results in the error).  If there is already a `file.json` in the current directory and I specify `--file=../file.json`, then the other file should still be copied into place.

Reply:
Ah. Fair enough. I'll update the PR comment to match the intent and merge. Thanks!